### PR TITLE
Fix doubled issue tab introduced in migration v16

### PR DESCRIPTION
**Pull request by daviian**
Migration v16 creates UnitTypeIssue and UnitTypeExternalTracker for each repository with enabled external issue tracker as repo.EnableIssues is true even for external tracker.

And because both Unit Types exist the issue tab is shown twice.

The same behaviour is happening to UnitTypeWiki and UnitTypeExternalWiki, but because the tab is not changed in view it has no visual impact.

PR fixes both mistakes and removes UnitTypeIssue for repositories with UniTypeExternalTracker and removes UnitTypeWiki for repositories with UnitTypeExternalWiki.

### CODE REVIEW

File: models/migrations/v44.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+		UnitTypeIssues                     // 2 issues
+		UnitTypePullRequests               // 3 PRs
+		UnitTypeReleases                   // 4 Releases
+		UnitTypeWiki                       // 5 Wiki
+		UnitTypeExternalWiki               // 6 ExternalWiki
+		UnitTypeExternalTracker            // 7 ExternalTracker
+	)
+
+	var externalIssueRepos []Repo
+	err := x.Table("repository").Select("`repository`.id").
```
Comment (lafriks) [inline models/migrations/v44.go]: No need for repository table here. Just select form repo_unit directly

File: models/migrations/v44.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+		Join("INNER", "repo_unit", "`repo_unit`.repo_id = `repository`.id").
+		Where("`repo_unit`.type = ?", UnitTypeExternalTracker).
+		Find(&externalIssueRepos)
+	if err != nil {
+		return fmt.Errorf("Query repositories: %v", err)
+	}
+
+	var externalWikiRepos []Repo
+	err = x.Table("repository").Select("`repository`.id").
```
Comment (lafriks) [inline models/migrations/v44.go]: Same as above

**Review by lafriks — CHANGES_REQUESTED** [review]

**Comment by daviian** [comment]
@lafriks Included your requested changes

**Review by lafriks — APPROVED** [review]

**Comment by lafriks** [comment]
LGTM

**Comment by Morlinest** [comment]
LGTM :)

**Comment by lunny** [comment]
LGTM

**Comment by lafriks** [comment]
Make LG-TM work

**Pull request merged**